homeassistant/components/homematicip_cloud/button.py

```diff
@@ -0,0 +1,41 @@
+"""Support for HomematicIP Cloud button devices."""
+from __future__ import annotations
+
+from homematicip.aio.device import AsyncWallMountedGarageDoorController
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import DOMAIN as HMIPC_DOMAIN, HomematicipGenericEntity
+from .hap import HomematicipHAP
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the HomematicIP button from a config entry."""
+    hap = hass.data[HMIPC_DOMAIN][config_entry.unique_id]
+    entities: list[HomematicipGenericEntity] = []
+    for device in hap.home.devices:
+        if isinstance(device, AsyncWallMountedGarageDoorController):
+            entities.append(HomematicipGarageDoorControllerButton(hap, device))
+
+    if entities:
+        async_add_entities(entities)
+
+
+class HomematicipGarageDoorControllerButton(HomematicipGenericEntity, ButtonEntity):
```

Comment:
The garage door controller has no state and no other controls than a single button, is that correct?

Reply:
Yes thats right. Just a simple button. No states.